packages/jest-get-type/src/isPrimitive.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import getType from './getType';
+
+const PRIMITIVES = new Set([
```

Comment:
Could type this as `Set<ValueType>` for compile-time checking that there's no values in it that `getType` will never actually return :sunglasses:

Reply:
https://github.com/facebook/jest/pull/7708/commits/eb297e788f254b4788162580a5b038f8920086f4